test/e2e/software_upgrade_e2e_v3_test.go

```diff
@@ -0,0 +1,221 @@
+package e2e
+
+import (
+	"math/rand"
+	"time"
+
+	"github.com/btcsuite/btcd/btcec/v2"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/stretchr/testify/suite"
+
+	appparams "github.com/babylonlabs-io/babylon/v3/app/params"
+	v3 "github.com/babylonlabs-io/babylon/v3/app/upgrades/v3"
+	"github.com/babylonlabs-io/babylon/v3/testutil/datagen"
+	"github.com/babylonlabs-io/babylon/v3/testutil/sample"
+	btclighttypes "github.com/babylonlabs-io/babylon/v3/x/btclightclient/types"
+	bstypes "github.com/babylonlabs-io/babylon/v3/x/btcstaking/types"
+	"github.com/babylonlabs-io/babylon/v3/x/finality/types"
+
+	"github.com/babylonlabs-io/babylon/v3/test/e2e/configurer"
+	"github.com/babylonlabs-io/babylon/v3/test/e2e/configurer/chain"
+	"github.com/babylonlabs-io/babylon/v3/test/e2e/configurer/config"
+)
+
+const (
+	btcstkconsumerModulePath = "btcstkconsumer"
+	zoneconciergeModulePath  = "zoneconcierge"
+)
+
+type SoftwareUpgradeV3TestSuite struct {
+	suite.Suite
+
+	configurer            *configurer.UpgradeConfigurer
+	balancesBeforeUpgrade map[string]sdk.Coin
+	fp1Addr               string
+	fp2Addr               string
+	r                     *rand.Rand
+	fp1BTCSK              *btcec.PrivateKey
+	fp2BTCSK              *btcec.PrivateKey
+	fp1                   *bstypes.FinalityProvider
+	fp2                   *bstypes.FinalityProvider
+	fp1CommitPubRandList  *types.MsgCommitPubRandList
+	fp2CommitPubRandList  *types.MsgCommitPubRandList
+	commitStartHeight     uint64
+}
+
+func (s *SoftwareUpgradeV3TestSuite) SetupSuite() {
+	s.T().Skip("Temporarily skipping v3 upgrade test")
```

Comment:
Was this skip on purpose? only noticed while doing https://github.com/babylonlabs-io/babylon/pull/1473